src/ucp/api/ucp.h

```diff
@@ -190,6 +190,15 @@ enum ucp_worker_params_field {
 };
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief UCP rkey comparison parameters field mask.
+ */
+enum ucp_rkey_compare_params_field {
+    UCP_RKEY_COMPARE_PARAM_FIELD_MODE = UCS_BIT(0) /**< Comparison mode */
```

Comment:
do we need mode in the initial implementation? since only one mode is supported now
maybe not define mode now, and if needed - extend it in the future with field_mask

Reply:
kept only `field_mask`, removed `enum ucp_rkey_compare_params_fields`